.github/CONTRIBUTING.md

```diff
@@ -24,8 +24,8 @@ To start contributing, fork the repository, create a new branch in your fork, ma
 
 Please make sure that your pull request contains unit tests covering what's being addressed by it.
 
-* All code should be compatible with PHPCS > 2.3.0 and PHPCS > 3.0.2.
-* All code should be compatible with PHP 5.3 to PHP nightly.
+* All code should be compatible with PHPCS > 2.6.0 and PHPCS > 3.0.2.
```

Comment:
>= 2.6.0 ?

Reply:
Good point. Fixed up in the relevant commit.